docs/user-guide/build-environment.md

```diff
@@ -11,4 +11,14 @@
 * `ARGOCD_APP_SOURCE_REPO_URL` the repo's URL
 * `ARGOCD_APP_SOURCE_TARGET_REVISION` - the target revision from the spec, e.g. `master`.
 * `KUBE_VERSION` - the version of kubernetes
-* `KUBE_API_VERSIONS` = the version of kubernetes API
\ No newline at end of file
+* `KUBE_API_VERSIONS` = the version of kubernetes API
+
+In case you don't want a variable to be interpolated, `$` can be escaped via `$$`.
+
+```
+command:
+  - sh
+  - -c
+  - |
+    echo $$FOO
```

Comment:
I don't know if this is a good example. I personally ran into this issue with a custom plugin but maybe a helm example would be better?